kolibri/plugins/learn/assets/src/vue/toolbar/index.vue

```diff
@@ -50,19 +49,20 @@
     },
     computed: {
       /*
-      * Get a list of channels.
+      * Get and set the current channel ID.
       */
-      getChannels() {
-        return this.channelList;
-      },
-      /*
-      * Get the current channel ID.
-      */
-      getCurrentChannel() {
-        return this.currentChannel;
+      getCurrentChannel: {
```

Comment:
while we're in here, maybe change from `getCurrentChannel` to just `currentChannel`?